widget/view.controller.js

```diff
@@ -472,7 +475,7 @@
               }
               $q.all(promises).then(function () {
                 var sortedDataSource = sortObjectByKeys(updatedObject);
-                sortedDataSource = Object.fromEntries(Object.entries(sortedDataSource).slice(0,3));
+                sortedDataSource = Object.fromEntries(Object.entries(sortedDataSource).slice(0, 3));
```

Comment:
Good practice: add comment why do you need slice with an example?

Reply:
Done